lib/customer.rb

```diff
@@ -0,0 +1,50 @@
+require_relative 'restaurant'
+
+class Customer
+  attr_reader :name, :order, :restaurant
+
+  def initialize(restaurant = Restaurant.new)
+    @restaurant = restaurant
+    @order = Hash.new(0)
+  end
+
+  def menu
+    restaurant.view_menu
+  end
+
+  def place_order(dish, quantity = 1)
+    raise "Please select a dish from the menu." if !restaurant.menu_include?(dish)
+    order[dish] += quantity
+    return "Thank you for ordering."
+  end
+
+  def review_order
+    raise "You haven't ordered anything." if order.empty?
```

Comment:
Style/SignalException: Use `fail` instead of `raise` to signal exceptions. (https://github.com/bbatsov/ruby-style-guide#fail-method)